### ISSUE TITLE
"Try it out" not working on IE11

### ISSUE THREAD

**Issue by darkylmnx** [babel/website#1627]
On the website, the page "try it out" does not work on IE11

http://babeljs.io/repl/

![image](https://user-images.githubusercontent.com/1426357/38811463-d764ed94-4189-11e8-9cfc-d9c88e82cfc1.png)

**Comment by xtuc** [comment]
Yeah Promise is not available on IE11. We should probably use `babel-preset-env`, and more likely on the new website.

**Comment by hzoo** [comment]
Same issue as https://github.com/babel/babel/issues/7607 I believe, we aren't using transform runtime or we should add the polyfill

**Comment by RusinovAnton** [comment]
I'm currently working on the fix. Its pretty trivial for the old site but in the new site there is another error arise:
```
Object doesn't support property or method 'forEach'
```
Which is forEach method of NodeList.  
The work on polyfill for this is in progress:
https://github.com/zloirock/core-js/pull/325

**Comment by hzoo** [comment]
Should be ok

**Comment by CoeJoder** [comment]
Does not work; getting same error.  IE 11.

**Comment by balzzac** [comment]
Try it out doesn't work, I get the same error in Chrome Canary Version 74.0.3698.0 (Official Build) canary (64-bit) and Chrome Version 72.0.3626.96 (Official Build) (64-bit)

**Comment by TrevorBurnham** [comment]
I believe this issue should be reopened. Currently, loading the "Try it out" page in IE11 yields a blank page (apart from the header) and these console errors:

<img width="343" alt="Screen Shot 2019-11-27 at 9 41 42 AM" src="https://user-images.githubusercontent.com/224895/69732795-442eaf00-10fa-11ea-80f9-5036b4396cb0.png">
